### ISSUE TITLE
How to redirect properly

### ISSUE THREAD

**Issue by jonquixote** [meteor-useraccounts/core#215]
I have been working on configuring routes for signin and signout
when I add

AccountsTemplates.configureRoute('signIn', {
    name: 'signin',
    redirect: '/home',
});

AccountsTemplates.configureRoute('signUp', {
    name: 'signUp',
    redirect: '/editprofile',
});

the redirects work fine and so do the routes and links. the problem is that with these (and a dozen similar) configurations, the user is first redirected to a new route with atForm in its default state. I tried to create a template with the state defined as signUp, but it would still load the default.

so basically with this configuration you need to click signup or sign in twice (one to load new route, one more to actually change state). this is annoying.

the one thing that I need out of this is for the sign in and register buttons to redirect the user to another page. if I can do this another way that would be great. for now this seems to be a bug...
